src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -15,8 +15,9 @@
     SupplierAddressId INT NULL,
     SupplierContactId INT NULL,
     CatalogueSolutionsViewed BIT DEFAULT 0,
-    AdditionalServicesViewed BIT DEFAULT 0,    
+    AdditionalServicesViewed BIT DEFAULT 0,
     AssociatedServicesViewed BIT DEFAULT 0,
+    FundingSourceViewed BIT DEFAULT 0,

```

Comment:
Funding source is more like commencement date. We don't have logic around when it's viewed, they have to choose yes or no.
I think it should be a nullable bit field, maybe 'FundingSourceOnlyGMS' or similar to reflect the question.